src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
```

Comment:
Good job! Try to rewrite your solution with `this`
```suggestion
    add(number) {
      this.result += number;
    },
```